src/layouts/sidebar-sidecar-with-toc/index.tsx

```diff
@@ -6,17 +6,16 @@
 import SidebarSidecarLayout, {
 	SidebarSidecarLayoutProps,
 } from 'layouts/sidebar-sidecar'
-import TableOfContents, {
-	TableOfContentsHeading,
-} from 'components/table-of-contents'
+import { TableOfContentsHeading } from 'components/table-of-contents'
 import { filterTableOfContentsHeadings } from 'components/table-of-contents/utils/filter-table-of-contents-headings'
+import { OutlineNavFromHeadings } from 'components/outline-nav/components'
 
 export type SidebarSidecarWithTocProps = SidebarSidecarLayoutProps & {
 	headings: TableOfContentsHeading[]
```

Comment:
I don't think so - but I think it's something we'd like to do eventually. It would involve refactoring `getStaticProps` for every page that uses this layout to change what's being returned from the server (currently `TableOfContentsHeading`, we probably want it to be `OutlineHeading`). This felt like more of a DX change than anything else, and the effort involved seems non-trivial, so I have a separate follow-up task 🎟 [Clean up headings-related OutlineNav uses, to move formatting closer to server](https://app.asana.com/0/1203810296555970/1204024966433303/f) to tackle it.

Let me know if that's helpful! 🙇